include/s3select_oper.h

```diff
@@ -569,7 +576,12 @@ class value
     value & operator^(const value &v)
     {
         return compute<binop_pow>(*this,v);
-    }
+    } 
+
+    value & operator%(const value &v)
+    {
+        return compute<binop_modulo>(*this,v);
```

Comment:
in the case of modulo , its an integer operation
the binop_modulo is implicitly casting to int.
user should be warn for wrong use of modulo operation (type is not integer)